nuitka/plugins/standard/ImplicitImports.py

```diff
@@ -259,6 +259,11 @@ def getImplicitImports(self, module):
             yield "apt_pkg", True
         elif full_name == "numpy.core":
             yield "numpy.core._dtype_ctypes", False
+        elif full_name == "scipy.special":
+            yield "scipy.special._ufuncs_cxx", False
```

Comment:
The `False` is really good for things that are not always that way. We don't explain that anywhere, do you?